freestyle/shared/src/main/scala/freestyle/free.scala

```diff
@@ -142,23 +142,23 @@ object freeImpl {
           import _root_.cats.free.Inject
           import _root_.freestyle.{ FreeS, FSHandler}
 
-          sealed trait $OP[$AA] extends scala.Product with java.io.Serializable
+          sealed trait Op[$AA] extends scala.Product with java.io.Serializable
```

Comment:
I think we should just use the string "Op" because it's a local name; the variable OP can be removed.